cranelift/codegen/src/isa/x64/inst/args.rs

```diff
@@ -736,10 +728,7 @@ impl fmt::Display for AluRmiROpcode {
 impl AluRmiROpcode {
     /// Is this a special-cased 8-bit ALU op?
     pub fn is_8bit(self) -> bool {
-        match self {
-            AluRmiROpcode::And8 | AluRmiROpcode::Or8 => true,
-            _ => false,
-        }
+        false
```

Comment:
Can this go away completely now (i.e. constant-prop the `false` outward to wherever it's called, if anywhere)?

Reply:
Definitely! I was a little unsure if removing `Add8` and `Or8` was desirable, so I didn't push the refactoring as far as I could have :)

Reply:
We likely won't need them for anything else I think -- the actual 8-bit ops use wider opcodes because it's more efficient microarchitecturally (no partial-register stalls) and the upper bits are don't-cares; at least `or8` was only needed here because we used SETcc which only sets the lower 8. We can always add them back if we find they're needed!